data/scripts.js

```diff
@@ -3592,7 +3594,25 @@ exports.BattleScripts = {
 			}
 
 			let monotypeRandomBattleSets = {
+				// Electric
```

Comment:
I know the sets will apply to both types at the moment. Sorting like this so I can easily change that later.